Source/astcenc_compress_symbolic.cpp

```diff
@@ -1221,8 +1221,8 @@ float compress_symbolic_block(
 
 	imageblock *temp = &tmpbuf->temp;
 
-	float best_errorvals_in_modes[17];
-	for (int i = 0; i < 17; i++)
+	float best_errorvals_in_modes[13];
+	for (int i = 0; i < 13; i++)
```

Comment:
magic numbers (13)

Reply:
For now I will leave this as the "minimal" edit to the existing code. I want to refactor this shortly; few of the values are actually used.